src/drivers/driver_zephyr.c

```diff
@@ -2063,6 +2063,29 @@ int wpa_drv_hapd_send_eapol(void *priv, const u8 *addr, const u8 *data, size_t d
 	return ret;
```

Comment:
The `Fixes SHEL-2681.` seems to be redundant in the commit message as it is only referring to Nordic internal systems.

Reply:
Yes, it was cherry-picked without manual changes,  I can remove that if needed.

Reply:
removed, also removed `nrf` from the sauce-tag.